client/proxy-lib.h

```diff
@@ -86,8 +86,8 @@ class ProxySnapperLib : public ProxySnapper
 
 public:
 
-    ProxySnapperLib(const string& config_name)
-	: snapper(new Snapper(config_name, "/")), proxy_snapshots(this)
+    ProxySnapperLib(const string& config_name, const string& target_root)
```

Comment:
this is just some internal class? I miss any documentation here. At least link to what it proxy.